sql/2022/javascript/frameworks-bytes-by-framework.sql

```diff
@@ -0,0 +1,35 @@
+#standardSQL
```

Comment:
Nit: this file name uses hyphen delimiters while most others use underscores. Change to underscores for consistency.